src/bin/parser-test.rs

```diff
@@ -1,12 +1,10 @@
-use gosub_engine::{
-    testing::{
-        self,
-        tree_construction::{ErrorResult, NodeResult, SubtreeResult, Test, TestResult},
-    },
-    types::Result,
-};
+use gosub_engine::testing::tree_construction::fixture::read_fixtures;
+use gosub_engine::testing::tree_construction::result::ResultStatus;
+use gosub_engine::testing::tree_construction::Harness;
+use gosub_engine::testing::tree_construction::Test;
```

Comment:
Not sure this is an improvement, but I've also seen some Rust code like it, so it's fine.

Reply:
Are you now talking about the fact that we have multiple structs / functions?